go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/stefanprodan/timoni
 go 1.22.0
 
 require (
-	cuelang.org/go v0.9.2
+	cuelang.org/go v0.11.0-alpha.1.0.20240910060539-63413629e06b
```

Comment:
We'll need to wait for v0.11.0 to go GA before merging.

Reply:
Yes... only if one could use build tags in 'go.mod'!